test/context.ts

```diff
@@ -1,21 +1,26 @@
 import Router from '@koa/router';
-import dataFactory from '@rdfjs/data-model';
 import { UnknownError } from 'http-errors';
-import { JsonLdObj } from 'jsonld/jsonld-spec';
 import Koa, { Context } from 'koa';
 import Request from 'koa/lib/request';
 import Response from 'koa/lib/response';
 import { Request as IncomingMessage, Response as ServerResponse } from 'mock-http';
+import { DatasetCore } from 'rdf-js';
 import InMemoryArticles from '../src/adaptors/in-memory-articles';
 import { AppContext } from '../src/app';
 import Articles from '../src/articles';
+import dataFactory, { dataset } from '../src/data-factory';
+import { WithDataset } from '../src/middleware/dataset';
 
 export type ErrorListener = (error: UnknownError, context: Context) => void;
 
+export type Headers = Record<string, string>;
+
 type Options = {
```

Comment:
the name `Options` seem to have become too generic, even within a `context` module like this? Most of the keys seem HTTP- or routing-related but not all of them (`articles`) which are domain-specific instead.

Reply:
Agree, but pre-existing so should change separately.